packages/core/src/api.js

```diff
@@ -115,6 +115,10 @@ export function createPercyServer(percy, port) {
     .route('post', '/percy/flush', async (req, res) => res.json(200, {
       success: await percy.flush(req.body).then(() => true)
     }))
+  // creating driverWrapper.js
+    .route('post', '/percy/automateScreenshot', async (req, res) => res.json(200, {
```

Comment:
`automateScreenshot` should we keep it in camel case instead we can call it on `screenshot` as well just saying because `url's` in general don't have camel casing.

Reply:
Let's rename it.